tests/test-class-wpseo-metabox.php

```diff
@@ -91,6 +91,69 @@ public function test_column_heading_has_metadesc() {
 	}
 
 	/**
+	 * Tests that column_hidden returns the columns if the option isn't set
```

Comment:
I assume you're testing here that the columns indeed are hidden? If so, please clarify what you're testing in the inline documentation.